### PR TITLE
add schemas to support attaching an exact array of vestings to any issuances with vesting (as an option to replace the need for vesting_terms_id)

### LINKED ISSUE #509: [Enhancement]: Enable a "Manual" Vesting Schedule Attached Directly to Issuances
## Description of Enhancement :

We've been doing more work with export and import, particularly of equity compensation and vesting schedules. Folks are finding some of the most complex issuances are bespoke and require that we generate vesting schedules from them only to then trust that the imported will perfectly import them on the other side. We think it would be safer and easier to allow someone to just create one-off or adhoc schedules with list of dates and vesting amounts that can be attached directly to issuances. 

### Why is this Needed?

Reduce data bloat and cut down on # of times vesting schedules need to be translated from data and amounts to OCF vesting schedule objects. 

### Anything else we need to know?

Nope

### PR DESCRIPTION
**Pull request by jacobyavis**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

To allow an implementer of OCF to forego the VestingTerms schema in favour of a simpler "manual" style schedule (a simple list of dates and amounts)

#### Which issue(s) this PR fixes:

Fixes #509

### CODE REVIEW

File: schema/objects/transactions/issuance/EquityCompensationIssuance.schema.json [hunk 1]
```diff
@@ -58,9 +58,18 @@
       "$comment": "REQUIRED in v2"
     },
     "vesting_terms_id": {
-      "description": "Identifier of the VestingTerms to which this security is subject.  If not present, security is fully vested on issuance.",
+      "description": "Identifier of the VestingTerms to which this security is subject. If neither `vesting_terms_id` or `vestings` are present then the security is fully vested on issuance.",
```
Comment (JSv4) [inline schema/objects/transactions/issuance/EquityCompensationIssuance.schema.json:61]: Now that we have the validator project, when we write natural language rules like this, let's also open a matching issue in the validator repo and cross-link the issues. That way we can finally start tracking these sorts of things. Even if it takes some time to get the rule into the XState code, we'll be able to keep track.

**Comment by JSv4** [comment]
Thanks for putting this together @jacobyavis

File: schema/objects/transactions/issuance/EquityCompensationIssuance.schema.json (hunk 1 above)
Comment (jacobyavis) [reply schema/objects/transactions/issuance/EquityCompensationIssuance.schema.json:61]: Good idea.
opened https://github.com/Open-Cap-Table-Coalition/OCF-XState-Validation/issues/76

**Review by JSv4 — APPROVED** [review]
LGTM

**Pull request merged**